src/components/calcite-button/calcite-button.tsx

```diff
@@ -150,12 +134,19 @@ export class CalciteButton {
 
     return (
       <Tag
-        {...attributes}
+        aria-label={this.el.getAttribute("aria-label")}
         class={{ [CSS_UTILITY.rtl]: dir === "rtl", [CSS.contentSlotted]: this.hasContent }}
         disabled={this.disabled}
+        href={this.childElType === "a" && this.href}
+        name={this.childElType === "button" && this.el.getAttribute("name")}
         onClick={this.handleClick}
         ref={(el) => (this.childEl = el)}
+        rel={this.childElType === "a" && this.el.getAttribute("rel")}
+        role={this.el.getAttribute("role")}
         tabIndex={this.disabled ? -1 : null}
+        target={this.childElType === "a" && this.el.getAttribute("target")}
+        title={this.el.getAttribute("title")}
```

Comment:
I don't think this is needed to be passed. title is a native attribute that can just be set on the host if needed.

Reply:
I'll try removing that and see if the accessibility tests still pass, I thought I saw one fail because the title wasn't being added, but I could be wrong.

Reply:
Yep, this is in fact needed by `calcite-fab`, which passes a `title` attribute to button.  Should I make `title` an official prop also?

Reply:
@eriklharper no, title is already a global property. It shouldn't need to be passed at all. Passing would make the title tooltip display on both nodes redundantly.

Reply:
Does this test need changed or does calcite-fab need to be changed as a result of this change then, because this test definitely isn't passing as a result of removing `title` 
![image](https://user-images.githubusercontent.com/821864/123718281-9a625780-d833-11eb-9b66-96bad6bf911c.png)

Reply:
I'm wondering what the difference is between `calcite-fab`'s usage of `aria-label` versus `title`?

Reply:
title wouldn't break a11y. Its probably how the aria-label is being set.

Reply:
Looks like title is breaking a11y in this case, otherwise it wouldn't say "Element has no title attribute" as one of the possible issues.  I'll try and see how the aria-label is being set to see if there's a way to fix that instead of adding the title.

Reply:
Just updated the test to pass both a label and a text attribute, that was the issue all along it appears.